runtime/jcl/exports.cmake

```diff
@@ -687,6 +687,13 @@ if(J9VM_OPT_CRIU_SUPPORT)
 	endif()
 endif()
 
+# J9VM_OPT_JFR
```

Comment:
I don't think this comment is helpful; I suggest the condition is obvious from the next line.

Reply:
So you suggest removing that line?

Reply:
Yes,